wiki/Beatmap/Guest_difficulty/en.md

```diff
@@ -11,6 +10,18 @@ tags:
 
 *For regulations surrounding guest difficulties, see: [Ranking Criteria](/wiki/Ranking_Criteria)*
 
-A **guest difficulty** (or *GD* for short) is a [difficulty](/wiki/Beatmap/Difficulty) of a [beatmap](/wiki/Beatmap) that is not created by the [beatmap host](/wiki/Beatmap/Beatmap_host). They can usually be identified by their difficulty name containing the guest creator's username. A beatmap featuring multiple creators is also known as a [collaborative difficulty](/wiki/Beatmap/Beatmap_collaborations).
+A **guest difficulty** (or *GD* for short) is a [difficulty](/wiki/Beatmap/Difficulty) of a [beatmap](/wiki/Beatmap) that is not created by the [beatmap host](/wiki/Beatmap/Beatmap_host). They can usually be identified by their difficulty name containing the guest creator's username.
```

Comment:
> o i think i'll leave this as is since i feel like it's rather awkward to have that description omitted from the line

i suggested removing it to be consistent with other instances of acronyms, but i suppose something like this could work as well. i disagree with having parentheses if it fits fit the flow of the sentence anyway (most of the time)

so either what i suggested or this

```suggestion
A **guest difficulty*, or *GD* for short, is a [difficulty](/wiki/Beatmap/Difficulty) of a [beatmap](/wiki/Beatmap) that is not created by the [beatmap host](/wiki/Beatmap/Beatmap_host). They can usually be identified by their difficulty name containing the guest creator's username.
```

Reply:
I think your suggestion is missing a `*`.